.gitignore

```diff
@@ -8,7 +8,7 @@ log/
 ebin/
 deps/*
 rel/erl-dns
-zones.json
+priv/_*.json
```

Comment:
This change is so I can have local zone data for simulation purposes without committing it to git.